docs/contributing/Contributing-To-Ignite.md

```diff
@@ -22,6 +24,39 @@ node your-ignite-repo-clone/bin/ignite generate component OrderButton
 
 In addition, we have automated tests you can run with `yarn test`.
 
+### Making PRs
+
+We appreciate your contribution! To make it easier for us to review, please make sure to add a clear title and description that explains both what the change is, and why it's useful, if applicable.
```

Comment:
Would it be possible to maybe add examples of what's clear or what's not clear? Maybe a link to a useful article.

Reply:
I find that difficult to describe in a general topic such as this one. I wanted to leave it open-ended since we haven't had clear standards and just encourage contributors to think about this.

I was thinking that as we see specific issues coming up we can iterate on adding specifics here.

If you have specific recommendations here, I'd be happy to take a look.

Reply:
> I wanted to leave it open-ended since we haven't had clear standards and just encourage contributors to think about this.

I agree with this. 

----

I did search for some merged PRs and sorted them by most commented but could not find a good example. 

I'm okay with leaving it open-ended and with no clear example. 

But one last suggestion 😅

What about adding a suggestion to browse [merged PRs](https://github.com/infinitered/ignite/pulls?q=is%3Apr+is%3Aclosed) so the contributor can get a sense of what has been done and how to potentially write a good PR description?

Reply:
Since you mentioned not finding good examples among the merged PRs, I think sending contributors to browse them might actually do the opposite of what we want and encourage a lack of description. I still think it might be better to keep things open-ended for now and iterate based on specific cases as they come up. Does that seem fair?

Reply:
Fair enough 🤝